### PR TITLE
document home-manager usage better

### PR DESCRIPTION
**Pull request by NotAShelf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,22 +21,53 @@ Config(
...
+        hm-options = inputs.home-manager.packages.${pkgs.system}.docs-json + "/share/doc/home-manager/options.json";
+        # merge your options
+        options = builtins.toJSON ["${nixos-options}"];
+        # or alternatively if you wish to read any other documentation options, such as home-manager
+        # options = builtins.toJSON ["${nixos-options}" "${nixos-options}" "${some-other-option}" /* ... */];
+
+    in ''
+        Config(
+            # add your option paths
+            options_path: "${options}",
```
Comment (n3oney) [inline README.md]: no "" in here cus you dont want the array wrapped in quotes

File: README.md [hunk 2]
```diff
@@ -21,22 +21,53 @@ Config(
...
+{inputs, pkgs, osConfig,  ...}: {
+    programs.anyrun.extraConfigFiles."nixos-options.ron".text = let
+        #               ↓ home-manager refers to the nixos configuration as osConfig
+        nixos-options = osConfig.system.build.manual.optionsJSON + "/share/doc/nixos/options.json";
+        # get the docs-json package from the home-manager flake
+        hm-options = inputs.home-manager.packages.${pkgs.system}.docs-json + "/share/doc/home-manager/options.json";
+        # merge your options
+        options = builtins.toJSON ["${nixos-options}"];
+        # or alternatively if you wish to read any other documentation options, such as home-manager
+        # options = builtins.toJSON ["${nixos-options}" "${nixos-options}" "${some-other-option}" /* ... */];
```
Comment (n3oney) [inline README.md]: i think you meant the second `nixos-options` to be `hm-options`

File: README.md [hunk 3]
```diff
@@ -21,22 +21,53 @@ Config(
...
+```nix
+#                  ↓ make sure osConfig is in the argument set
+{inputs, pkgs, osConfig,  ...}: {
+    programs.anyrun.extraConfigFiles."nixos-options.ron".text = let
+        #               ↓ home-manager refers to the nixos configuration as osConfig
+        nixos-options = osConfig.system.build.manual.optionsJSON + "/share/doc/nixos/options.json";
+        # get the docs-json package from the home-manager flake
+        hm-options = inputs.home-manager.packages.${pkgs.system}.docs-json + "/share/doc/home-manager/options.json";
+        # merge your options
+        options = builtins.toJSON ["${nixos-options}"];
```
Comment (n3oney) [inline README.md:54]: is the "${}" actually necessary?

**Review by n3oney — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (NotAShelf) [reply README.md]: what do you mean, I clearly didn't

File: README.md (hunk 3 above)
Comment (NotAShelf) [reply README.md:54]: I don't really know, haven't tested but it does not hurt

File: README.md [hunk 4]
```diff
@@ -21,22 +21,53 @@ Config(
...
+        # or alternatively if you wish to read any other documentation options, such as home-manager
+        # get the docs-json package from the home-manager flake
+        # hm-options = inputs.home-manager.packages.${pkgs.system}.docs-json + "/share/doc/home-manager/options.json";
+        # options = builtins.toJSON ["${nixos-options}" "${hm-options}" "${some-other-option}" /* ... */];
+
+    in ''
+        Config(
+            # add your option paths
+            options_path: ${options},
```
Comment (n3oney) [inline README.md]: path*s*

**Review by n3oney — CHANGES_REQUESTED** [review]

**Comment by NotAShelf** [comment]
ok done

**Comment by n3oney** [comment]
could u squash commits tho:)

**Comment by n3oney** [comment]
nvm i can do it lol

**Comment by n3oney** [comment]
thank you

**Pull request merged**